.github/workflows/php-test-plugins.yml

```
@@ -68,6 +72,32 @@ jobs:
       - name: Composer Install
         run: npm run wp-env run tests-cli -- --env-cwd="wp-content/plugins/$(basename $(pwd))" composer install --no-interaction --no-progress
       - name: Running single site unit tests
-        run: npm run test-php
+        run: |
+          if [ "${{ matrix.coverage }}" == "true" ]; then
+            npm run test-php -- --coverage-clover=coverage.xml
```

Maybe append commit SHA to avoid chance of conflicts?
```suggestion
            npm run test-php -- --coverage-clover=coverage-${{ github.sha }}.xml
```

---

How would there be a conflict? Each GHA job should run from a pristine state, no? For the `coverage.xml` file to persist across runs this would require some explicit caching.

---

Probably. I find this safer, but either works for me.

---

> I find this safer.

I think better safe than sorry.
So implementing this as well. 😀